### DISCUSSION TITLE
No result from sponsorsActivities

### DISCUSSION THREAD

**Discussion by KoKin941112** [community/community#35289] [Other Feature Feedback, Questions, & Ideas] [answered]
I used the following query to access the sponsorsActivities of Microsoft but got an empty result. Microsoft indeed has sponsor activities.
Can anyone help me solve the problem? Thanks!

![image](https://user-images.githubusercontent.com/37288726/194209891-4c96b8d5-0eaa-43b4-ae16-90b2ade415ea.png)

**Comment by cheshire137** [comment, marked as answer]
Hi there! I'm a developer on the Sponsors team. The `sponsorsActivities` field limits the return values to just those involving you as the sponsor or the maintainer. So if you're not an admin or billing manager of `Microsoft`, or the recipient of a sponsorship from `Microsoft`, you wouldn't get back any results when querying their activities.

**Reply by KoKin941112** [reply]
Thanks!
